mm/2s2h/Rando/Logic/Regions/StoneTowerTemple.cpp

```diff
@@ -152,32 +152,40 @@ static RegisterShipInitFunc initFunc([]() {
         },
         .connections = {
             CONNECTION(RR_STONE_TOWER_TEMPLE_LAVA_WIND_ROOM, CAN_KILL_GARO_MASTER),
-            CONNECTION(RR_STONE_TOWER_TEMPLE_SPIKED_BAR_ROOM, CAN_KILL_GARO_MASTER)
+            CONNECTION(RR_STONE_TOWER_TEMPLE_SPIKED_BAR_ROOM_UPPER, CAN_KILL_GARO_MASTER)
         }
     };
-    Regions[RR_STONE_TOWER_TEMPLE_SPIKED_BAR_ROOM] = RandoRegion{ .name = "Spiked Bar Room", .sceneId = SCENE_INISIE_N,
+    Regions[RR_STONE_TOWER_TEMPLE_SPIKED_BAR_ROOM_LOWER] = RandoRegion{ .name = "Spiked Bar Room Lower", .sceneId = SCENE_INISIE_N,
+        .checks = {
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_1, true),
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_2, true),
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_3, true),
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_4, true),
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_5, true),
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_6, true),
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_7, true),
+            CHECK(RC_STONE_TOWER_TEMPLE_POT_BEFORE_WATER_BRIDGE_8, true),
```

Comment:
Since the region is called Spiked Bar Room, maybe these pots should get renamed to `RC_STONE_TOWER_TEMPLE_POT_SPIKED_BAR_ROOM_#`?

Reply:
Honestly I imagine many RCs will need to be renamed. But sure I can quickly rename those.